compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -4299,6 +4299,15 @@ private boolean isValidExprRhsStart(SyntaxKind tokenKind) {
         }
     }
 
+    private boolean isInvalidFunctionCallStart(SyntaxKind tokenKind, STNode lhsExpr) {
```

Comment:
Shouldn't this be merged with `isValidExprRhsStart`?
Because there are other places where we use `isValidExprRhsStart` for the same purpose, and those should behave the same way right?